src/desktop/apps/collect2/server.js

```diff
@@ -1,22 +1,30 @@
-import { buildServerApp } from 'reaction/Artsy/Router'
-import { renderLayout } from '@artsy/stitch'
-import { routes } from 'reaction/Apps/Collect/routes'
-import mediator from 'desktop/lib/mediator.coffee'
-import express from 'express'
-import React from 'react'
+import { buildServerApp } from "reaction/Artsy/Router"
+import { renderLayout } from "@artsy/stitch"
+import { routes } from "reaction/Apps/Collect/routes"
+import { Meta } from "./meta.tsx"
+import mediator from "desktop/lib/mediator.coffee"
+import express from "express"
+import React from "react"
+import getCollectPageTitle from "../../components/commercial_filter/page_title"
 
 const app = (module.exports = express())
 
 export const index = async (req, res, next) => {
   try {
     const user = req.user && req.user.toJSON()
+    const { APP_URL, COLLECT_PAGE_TITLES_URL, IS_MOBILE } = res.locals.sd
+
+    // Maybe get custom page title based on query data
+    const collectRawData = await fetch(COLLECT_PAGE_TITLES_URL)
+    const collectData = await collectRawData.json()
```

Comment:
That's a nice touch but I would say let's leave custom page titles out for now. 
The title changes dynamically client-side as well on the old collect page but since we have a different set of filters the added complexity doesn't make sense for now.